ros/install.bash

```diff
@@ -3,7 +3,14 @@
 
 if [ -z "$TUE_ROS_DISTRO" ]
 then
-    tue-install-error "TUE_ROS_DISTRO was not set"
+    echo "[tue ros install] TUE_ROS_DISTRO was not set"
+    echo "[tue ros install] Did you run the bootstrap-ros1.bash script?"
```

Comment:
We don't have this script...

Reply:
I'll remove it.

Probably the script will be introduced if we want a ROS2 migration backported :smile: